src/kernel-install/90-loaderentry.install

```diff
@@ -104,10 +104,11 @@ if ! [ -d "$ENTRY_DIR_ABS" ]; then
     exit 1
 fi
 
-install -g root -o root -m 0644 "$KERNEL_IMAGE" "$ENTRY_DIR_ABS/linux" || {
+install -m 0644 "$KERNEL_IMAGE" "$ENTRY_DIR_ABS/linux" || {
     echo "Error: could not copy '$KERNEL_IMAGE' to '$ENTRY_DIR_ABS/linux'." >&2
     exit 1
 }
+chown root.root "$ENTRY_DIR_ABS/linux" || :
```

Comment:
I'm not sure about suddenly silently failing when we can't change ownership. Should we do this at all?

Reply:
It's not silent. `chown` prints an error.  I think it's fine to do this. In practice this only matters when running as non-root, i.e. in some test scenario.

Reply:
In practice, these should almost always be written to fat, no? chown doesn't work on fat, so it'll always complain.